doc-deploy-dev/action.yml

```
@@ -50,14 +51,23 @@ runs:
       run: |
         rm -rf dev/ && mkdir dev/
 
-    - name: "Download the HTML documentation artifact"
+    - name: "Download the HTML documentation artifact from desired workflow"
       uses: dawidd6/action-download-artifact@v2
+      if: inputs.workflow != ''
       with:
         github_token: ${{ inputs.token }}
         name: ${{ inputs.doc-artifact-name }}
         path: dev
         workflow: ${{ inputs.workflow }}
 
+    - name: "Download the HTML documentation artifact from current workflow"
+      uses: dawidd6/action-download-artifact@v2
+      if: inputs.workflow != ''
+      with:
+        github_token: ${{ inputs.token }}
+        name: ${{ inputs.doc-artifact-name }}
+        path: dev
+
```

```suggestion

    - name: "Download the HTML documentation artifact from current workflow"
      uses: dawidd6/action-download-artifact@v2
      if: inputs.workflow == ''
      with:
        github_token: ${{ inputs.token }}
        name: ${{ inputs.doc-artifact-name }}
        path: dev

```
I am guessing this should be like this... but please correct me if I'm wrong

---

I changed the ``!=`` for ``==``

---

Yes, thanks @RobPasMue